tools/pipelines/test-azure-frs.yml

```diff
@@ -5,74 +5,148 @@
 
 name: $(Build.BuildId)
 
+schedules:
```

Comment:
Lets chat with Alejandro about using existing perf pipelines instead of extending test-azure-frs.